plugins/needs_restarting.py

```diff
@@ -35,6 +35,11 @@
 import stat
 
 
+# For which package updates we should recommend a reboot
+# Taken from https://access.redhat.com/solutions/27943
+NEED_REBOOT = ['kernel', 'glibc', 'linux-firmware', 'systemd', 'dbus']
```

Comment:
`dbus-broker` and `dbus-daemon` are also probably ones that belong on this list...

Reply:
Those are not mentioned explicitly on the KB page (only "dbus" is), but yeah, they seem to be quite essential so I'll include them. In any case, we should think about a more flexible way in the future as having a hard-coded list doesn't scale well :)

Reply:
Also, I'd like to revisit this plugin later as it deserves more love and could be more useful and user-friendly. But that's for later.

Reply:
Note to self: the plugin could also be renamed to "ps" for simplicity (and consistency with zypper).